src/Viz.js

```diff
@@ -55,7 +55,7 @@ const SUPPORTED_PROPERTIES = [
  * @property {Placement} symbolPlacement - when using `symbol`, offset to apply to the image
  * @property {Placement} offset - offset to apply to the features in pixels
  * @IGNOREproperty {Order} order - rendering order of the features, only applicable to points
- * @property {number} resolution - resolution of the property-aggregation functions, a value of 4 means to produce aggregation on grid cells of 4x4 pixels, only applicable to points
+ * @property {number} resolution - resolution, from 1 to 255, of the property-aggregation functions, a value of 4 means to produce aggregation on grid cells of 4x4 pixels, only applicable to points
```

Comment:
@jgoizueta can confirm, but I think that the minimum value is slightly above `0`, like `0.000001`

Reply:
Yes the backend supports less than 1 resolutions (https://github.com/CartoDB/Windshaft-cartodb/blob/master/docs/aggregation.md#resolution). (so that 0.5 would be a quadrant of a pixel).

The backend actually only checks that `resolution` is [greater than 0](https://github.com/CartoDB/Windshaft-cartodb/blob/204d246a8cf227bbfde85d7fd662a97bd73fd8c3/lib/cartodb/models/aggregation/aggregation-mapconfig.js#L69), but very small values would cause timeouts, so it would be better to have a explicit minimum.

Reply:
ok! I am going to change that :) thanks!